### ISSUE TITLE
To add updates of labels in PRs

### ISSUE THREAD

**Issue by xxxtonixxx** [gogits/gogs#2520]
When I remove or add labels doesn't show activity of them (if I remove or add one).

- For ex:
![image](https://cloud.githubusercontent.com/assets/7110786/12697853/6f2dd5ec-c78d-11e5-9df0-3e81bfa8a4d6.png)

Saludos and sorry for my bad english!

**Comment by Rick-Anderson** [comment]
@tonivj5 can you change the title to "show the history of issue label change" ? See #4854 which get a lot more traction. We want to make this more discoverable so it gets more up votes.

**Comment by tonivj5** [comment]
> @tonivj5 can you change the title to "show the history of issue label change" ? See #4854 which get a lot more traction. We want to make this more discoverable so it gets more up votes.

Yeah, I'm going to change the title :+1:. Btw, I don't see where or how #4854 has got more tranction than this issue, tbh


> I think #4854 should be reopened. I don't see this as a duplicate

What's the difference? I think it's exactly the same issue (as @unknwon labeled it)
